frontend/src/app/home/containers/Home.js

```diff
@@ -51,12 +51,27 @@ export default class Home extends React.Component {
 
   }
 
+  sellStock(stock, quantity, price){
```

Comment:
What is this?

Reply:
Function for selling owned stocks that are displayed on the home component